### DISCUSSION TITLE
When running it does not do anything

### DISCUSSION THREAD

**Discussion by CheesyPoofs3** [3kh0/ext-remover#47] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When I try to run a bookmarklet it will do nothing

**Comment by CheesyPoofs3** [comment]
when i run any bookmarklet is does nothing

**Comment by 3kh0** [comment, marked as answer]
Duplicate of #38

**Reply by CheesyPoofs3** [reply]
its not blocked though

**Reply by 3kh0** [reply]
cany ou find the URLBlocklist setting and send me a screenshot of it?
